contracts/strategies/rfp-simple/RFPSimpleStrategy.sol

```diff
@@ -407,12 +408,19 @@ contract RFPSimpleStrategy is BaseStrategy, ReentrancyGuard {
         // Update status of acceptedRecipientId to accepted
         recipient.recipientStatus = Status.Accepted;
 
+        if (recipient.proposalBid != finalProposalBid) {
+            // If the proposal bid is not equal to the final proposal bid this will revert
+            // This is to prevent the pool manager from decreasing the proposal bid
+            // or recipient from front running and increasing the proposal bid
+            revert AMOUNT_TOO_LOW();
```

Comment:
since there could be different reasons of why the proposal bids do not match, i would suggest to use a more generic error like `INVALID`